### ISSUE TITLE
Question about saving multiple images from large stream

### ISSUE THREAD

**Issue by ssharma0312** [Http-Multipart-Data-Parser/Http-Multipart-Data-Parser#65]
Hi Guys

Apologies to post this as in issue. It's more of question since I am novice. I have a large stream (real estate property photos), and using StreamingMultipartFormDataParser to parse. I am looking to find a c# sample code to how to save images to my sever after fileHandler step. Parser.filehandler. 

I am looking to loop through the files. But following code doesn't seem to work when using StreamingMultipartFormDataParser

```csharp
// Loop through all the files
foreach(var file in parser.Files)
{
    Stream data = file.Data;

    // Do stuff with the data.
}
```

**Comment by Jericho** [comment]
Did the files have the same name (or a blank name)? I'm asking because an issue was recently [reported](#81) when multiple files are uploaded with the same name. This issue caused their content to be combined into a single file and it was fixed in release 4.0.0. 

I suggest you upgrade to the latest nuget package and let us know if it takes care of he issue you experienced.

**Comment by Jericho** [comment]
I am going to assume the latest release resolved this issue but feel free to comment and reopen this issue if you find that it's not the case.
